### PR TITLE
chore: support link in settings page

### PR DESCRIPTION
**Pull request by bluecco**

### CODE REVIEW

File: packages/extension/src/ui/features/settings/SupportFooter.tsx [hunk 1]
```diff
@@ -1,86 +1,76 @@
...
+      flex="1"
+      gap={2}
+      p={4}
+      w="100%"
+      h={"initial"}
+      rounded={"lg"}
+      justifyContent={"center"}
+      {...rest}
+    >
+      <Flex fontSize="base">{leftIcon}</Flex>
```
Comment (simonheys) [inline packages/extension/src/ui/features/settings/SupportFooter.tsx:25]: `Button` has a `leftIcon` prop just for this purpose https://chakra-ui.com/docs/components/button#button-with-icon

File: packages/extension/src/ui/features/settings/SupportFooter.tsx [hunk 2]
```diff
@@ -1,86 +1,76 @@
...
+      <B3>{children}</B3>
```
Comment (simonheys) [inline packages/extension/src/ui/features/settings/SupportFooter.tsx:26]: Should be able to use `<Button size="sm" ... ` to get the correct font size here, or add another size variant if we don't have the right one yet

File: packages/extension/src/ui/features/settings/SupportFooter.tsx [hunk 3]
```diff
@@ -1,86 +1,76 @@
...
+        style={{ display: "flex", flex: 1 }}
         href="https://support.argent.xyz/hc/en-us/categories/5767453283473-Argent-X"
         title="Get ArgentX Support"
         target="_blank"
       >
-        <IconWrapper>
-          <SupportIcon />
-          <IconText>Support</IconText>
-        </IconWrapper>
+        <SupportButton leftIcon={<SupportIcon />}>Help</SupportButton>
```
Comment (simonheys) [inline packages/extension/src/ui/features/settings/SupportFooter.tsx:46]: Should be able to do `<Button as='a' ...` and include the href and other `a` props here

File: packages/extension/src/ui/features/settings/SupportFooter.tsx [hunk 4]
```diff
@@ -1,86 +1,76 @@
...
 
 const SupportFooter: FC = () => (
-  <Footer>
-    <P>Help, support &amp; suggestions:</P>
-    <RowCentered gap="10px">
+  <VStack mt={4} borderTop="solid 1px" borderTopColor="neutrals.700">
+    <P3 color="neutrals.400" pt="6">
+      Help, support &amp; suggestions:
+    </P3>
+    <Flex gap="2" w="100%" py={4}>
```
Comment (simonheys) [inline packages/extension/src/ui/features/settings/SupportFooter.tsx:39]: Can maybe use `SimpleGrid` with columns here - used it for same effect here https://github.com/argentlabs/argent-x/blob/65567a7fd6617bfa4d657b0873887488e41bc0d7/packages/extension/src/ui/features/accountTokens/AccountTokensButtons.tsx#L82

File: packages/extension/src/ui/features/settings/SettingsScreen.tsx [hunk 5]
```diff
@@ -142,8 +143,17 @@ export const SettingsScreen: FC = () => {
...
+
+          <SupportFooter />
+          <SpacerCell h="16" />
+        </CellStack>
+        <Box bg="neutrals.900" position="fixed" bottom="0" left="0" right="0">
           <Link onClick={stopSession} to={routes.lockScreen()}>
-            <SpacerCell />
+            <SpacerCell
+              borderTop="solid 1px"
+              borderTopColor="neutrals.700"
```
Comment (simonheys) [inline packages/extension/src/ui/features/settings/SettingsScreen.tsx:154]: For consistency there is a semantic color token "border" for border color, defined here https://github.com/argentlabs/argent-x/blob/65567a7fd6617bfa4d657b0873887488e41bc0d7/packages/ui/src/theme/semanticTokens.ts#L11

**Review by janek26 — APPROVED** [review]

**Pull request merged**